src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -130,8 +130,12 @@ disable_block_device_use = @DEFDISABLEBLOCK@
 # Shared file system type:
 #   - virtio-fs (default)
 #   - virtio-9p
+#   - virtio-nydus
```

Comment:
I think `virtio-fs-nydus` would be better, as there is not a virtio device type called nydus.

Reply:
agree